### PR TITLE
chore: update coc with include subject line

### PR DESCRIPTION
**Pull request by SethFalco**
I assume there's no need to update the copy in the root of the repository; I can see your bot takes care of that. 🤔 

Happy with this?

I just have 2 questions:

Have a preference between these two?
  * We will take your report seriously.
  * We will deal with all CoC issues as a priority. ← My preference.

I actually disagree with the use of the word "will" in this case. The word "will" usually denotes the willingness to do something, or the intent to do something in future. I assume you already take reports seriously and deal with CoC issues as a priority. The word "will" is out of place in my opinion.

In my opinion "We deal with all CoC issues as a priority." is much better. It states what you actually do rather than what you're willing to do, which may or may not be what do now. Thoughts on this?

On the other hand, I got an E in GCSE English so what do I know. 🥲  

---

### Related
* https://github.com/exercism/exercism/issues/6352

### CODE REVIEW

File: global-files/CODE_OF_CONDUCT.md
Comment (SaschaMann) [inline global-files/CODE_OF_CONDUCT.md:9]: I'd prefer `[abuse@exercism.org](mailto:abuse@exercism.org?subject=COC+violation)` or something along those lines.

Comment (SethFalco) [reply global-files/CODE_OF_CONDUCT.md:9]: Linking it is a good call! As for that specific suggestion, I'll wait for confirmation before doing that since that would require updating in other places as well then.

Comment (iHiD) [reply global-files/CODE_OF_CONDUCT.md]: Subject should stay as `[COC]` as there's stuff that filters those the other end, but agree with adding a link if we can 🙂

**Comment by iHiD** [comment]
I think "will" has an air of being reassuring in that "this will happen when you report". But I think the "will" then needs to be paired with a "you".

So let's change it to say "We will follow up with you as a priority" or similar.

File: global-files/CODE_OF_CONDUCT.md
Comment (SaschaMann) [reply global-files/CODE_OF_CONDUCT.md]: (I was too lazy to look up the encoding for those braces :p)

**Comment by SethFalco** [comment]
Done!
I used "follow-up" instead of "follow up", otherwise I just took your wording.

* https://www.merriam-webster.com/dictionary/follow-up
* https://www.wordnik.com/words/follow-up

**Review by SaschaMann — APPROVED** [review]
I'm fine with it but Erik should decide when and how to merge it.

**Comment by kytrinyx** [comment]
In this context I believe _follow up_ here acts as a phrasal verb rather than a compound noun. The compound noun takes a hyphen, but the phrasal verb does not.

Ref: http://www.enhancemywriting.com/followup-or-follow-up

**Comment by ErikSchierboom** [comment]
> I'm fine with it but Erik should decide when and how to merge it.

So, it depends on how important we feel this change is. We could merge this which would also fix the trailing whitespace on repos.

There are also still open org-wide-files PRs that have not been merged. I can either:

1. Merge them before merging this
2. Close them and merge this to have them be recreated

I don't have particularly strong opinions on this. I think I'll prefer merging the existing PRs first.

**Comment by SethFalco** [comment]
> In this context I believe follow up here acts as a phrasal verb rather than a compound noun. The compound noun takes a hyphen, but the phrasal verb does not.

Ahh, nice resource!

I appreciate the heads-up and source, I'll change it back.

**Review by iHiD — APPROVED** [review]

**Comment by iHiD** [comment]
Yep! :)

**Pull request merged**
